packages/headless/src/features/generated-answer/generated-answer-analytics-actions.ts

```diff
@@ -245,13 +246,16 @@ export const logGeneratedAnswerStreamEnd = (
     (client, state) => {
       const generativeQuestionAnsweringId =
         generativeQuestionAnsweringIdSelector(state);
+      const answerTextIsEmpty =
+        !state.generatedAnswer?.answer || !state.generatedAnswer?.answer.length;
       if (!generativeQuestionAnsweringId) {
         return null;
       }
       return client.makeGeneratedAnswerStreamEnd({
         generativeQuestionAnsweringId,
         answerGenerated,
-      });
+        answerTextIsEmpty,
+      } as GeneratedAnswerStreamEndMeta);
```

Comment:
Why is a cast needed here ? 🤔

Reply:
yes as I said: 
`open PR in Coveo.analytics to add this new parameter to the interface here: https://github.com/coveo/coveo.analytics.js/pull/452
In the meantine added a Type checker `

We will wait that it s deployed before merging this PR, and it will be removed then